### PR TITLE
Fix eth2 accounting processing

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/eth2/structures.py [hunk 1]
```diff
@@ -218,7 +218,7 @@ def process(
             accounting: 'AccountingPot',
             events_iterator: Iterator['AccountingEventMixin'],  # pylint: disable=unused-argument
     ) -> int:
-        if self.pnl_balance == ZERO:
+        if self.pnl_balance.amount == ZERO:
             return 1
 
         amount = self.pnl_balance.amount
```
Comment (yabirgb) [inline rotkehlchen/chain/ethereum/modules/eth2/structures.py:224]: You could have moved the amount variable declared before the if statement but is fine

Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/modules/eth2/structures.py:224]: hmm fuck yeah that would have been prettier. Oh well next PR :smile:

Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/modules/eth2/structures.py]: Actually changed it since more changes had to be added

**Comment by LefterisJP** [comment]
It's not easy being greeeeeeen, but when it is I need to meeeeerge

**Pull request merged**